project_multifamily_beta/housing_characteristics/Vintage ACS.tsv

```diff
@@ -1,1945 +1,11 @@
-10 to 19 Unit	MI_Muskegon.County.AP.726360	0.036447	0.049610	0.317118	0.375466	0.191285	0.030075	3865.062380
-10 to 19 Unit	MI_Sault.Ste.Marie-Sanderson.Field.727340	0.082566	0.041599	0.255377	0.440881	0.166584	0.012992	681.787788
-10 to 19 Unit	MI_Traverse.City-Cherry.Capital.AP.726387	0.036691	0.065365	0.248813	0.454552	0.189509	0.005071	2056.192812
-10 to 19 Unit	MN_Duluth.Intl.AP.727450	0.086774	0.054823	0.354402	0.373373	0.125821	0.004807	3720.798110
-10 to 19 Unit	MN_International.Falls.Intl.AP.727470	0.040667	0.047380	0.265542	0.416759	0.192711	0.036940	712.754277
-10 to 19 Unit	MN_Minneapolis-St.Paul.Intl.AP.726580	0.096060	0.101451	0.458692	0.267068	0.067635	0.009094	43573.366800
-10 to 19 Unit	MN_Rochester.Intl.AP.726440	0.034142	0.066460	0.376117	0.410493	0.108392	0.004396	3010.781246
-10 to 19 Unit	MN_St.Cloud.Muni.AP.726550	0.022646	0.031222	0.307024	0.536438	0.089099	0.013570	8916.542236
-10 to 19 Unit	MO_Columbia.Rgnl.AP.724450	0.019048	0.024852	0.272656	0.418443	0.225471	0.039529	6587.786583
-10 to 19 Unit	MO_Kansas.City.Intl.AP.724460	0.021811	0.047206	0.329522	0.410772	0.170082	0.020608	32411.046790
-10 to 19 Unit	MO_Springfield.Rgnl.AP.724400	0.007786	0.014428	0.139126	0.523207	0.288252	0.027202	17895.348570
-10 to 19 Unit	MO_St.Louis-Lambert.Intl.AP.724340	0.040004	0.056361	0.357599	0.414385	0.113801	0.017849	34396.964610
-10 to 19 Unit	MS_Jackson.Intl.AP.722350	0.005384	0.024160	0.383212	0.429496	0.137138	0.020610	7472.218830
-10 to 19 Unit	MS_Meridian-Key.Field.722340	0.009013	0.056278	0.195567	0.409581	0.250715	0.078845	2680.700903
-10 to 19 Unit	MT_Billings-Logan.Intl.AP.726770	0.088318	0.042732	0.276518	0.333685	0.183875	0.074872	1099.109388
-10 to 19 Unit	MT_Cut.Bank.Muni.AP.727796	0.027742	0.079430	0.490676	0.383747	0.018404	0.000000	252.789419
-10 to 19 Unit	MT_Glasgow.Intl.AP.727680	0.059297	0.185087	0.454637	0.150658	0.129654	0.020667	187.388512
-10 to 19 Unit	MT_Great.Falls.Intl.AP.727750	0.081143	0.095567	0.561352	0.162557	0.058557	0.040824	1143.308567
-10 to 19 Unit	MT_Helena.Rgnl.AP.727720	0.099099	0.150921	0.230309	0.233026	0.235428	0.051216	1960.376215
-10 to 19 Unit	MT_Kalispell-Glacier.Park.Intl.AP.727790	0.075798	0.070076	0.155004	0.414721	0.274236	0.010165	567.272505
-10 to 19 Unit	MT_Lewistown.Muni.AP.726776	0.120831	0.058869	0.691824	0.052488	0.024569	0.051419	313.390390
-10 to 19 Unit	MT_Miles.City.Muni.AP.742300	0.085000	0.186612	0.357388	0.245362	0.088302	0.037335	229.458476
-10 to 19 Unit	MT_Missoula.Intl.AP.727730	0.081277	0.021849	0.218162	0.346249	0.314104	0.018358	1927.240160
-10 to 19 Unit	NC_Asheville.Rgnl.AP.723150	0.041571	0.026256	0.142110	0.413630	0.290468	0.085964	4144.760806
-10 to 19 Unit	NC_Cape.Hatteras.723040	0.013871	0.015713	0.262946	0.472520	0.177186	0.057765	442.587195
-10 to 19 Unit	NC_Charlotte-Douglas.Intl.AP.723140	0.005872	0.016006	0.152317	0.553307	0.236891	0.035608	43903.026830
-10 to 19 Unit	NC_Greensboro-Piedmont.Triad.Intl.AP.723170	0.009318	0.026347	
```

Comment:
Do we still need Vintage ACS after this? If it is just for CHIA, then maybe we remove it eventually?

Reply:
We might be able to remove it eventually. Vintage ACS is helpful if we want to reduce the size of a tsv file when there are a large number of dependencies (ex: Heating Fuel) or when there are we are slicing a survey a lot and there are lower sample sizes.